stash/test/trades.test.ts

```diff
@@ -75,19 +77,19 @@ describe.skip('test trades processor', () => {
 
     // check len of the processed entries
     // event every 25 block
-    store.get(fixtures.asset_10.id)!.length.should.be.equal(fixtures.LEN / 25)
```

Comment:
Made to match the state, qa to check for the future.

Reply:
no clue about those :(